app/src/main/kotlin/co/netguru/baby/monitor/client/feature/babynotification/BabyMonitorMessagingService.kt

```diff
@@ -63,8 +66,8 @@ class BabyMonitorMessagingService : FirebaseMessagingService() {
         )
     }
 
-    private fun handleCryNotification(data: MutableMap<String, String>) {
-        database.getChildData()
+    private fun handleBabyEvent(data: MutableMap<String, String>) {
+        disposables += database.getChildData()
```

Comment:
In one class u're using `.addTo(disposables`, this time `disposables +=` try to keep same convention in whole project

Reply:
When i'm adding something new i always try to follow `disposable +=`. But i need to refactor all the `addTo`es .